### ISSUE TITLE
setState just works when Component render returns an array

### ISSUE THREAD

**Issue by janmarkuslanger** [janmarkuslanger/powerjs#11]

**Comment by janmarkuslanger** [comment]
This works but when it just returns the HTML Element it does not.
``` javascript 
class myTest extends Power.Component {
  render() {
    return [
      Power.h('h1', this.data.counter.toString(), {click: () => {
        this.setState(() => {
          this.data.counter += 1;
        })
      }})
    ]
  }
}
```

Not working:

´´´ javascript
class myTest extends Power.Component {
  render() {
    return
      Power.h('h1', this.data.counter.toString(), {click: () => {
        this.setState(() => {
          this.data.counter += 1;
        })
      }})
  }
}
```

**Comment by janmarkuslanger** [comment]
Fix in 1.3.2
